### PR TITLE
Add deploy button and fix instructions for AWS IoT Integration

### PR DESCRIPTION
**Pull request by rahul-thakoor**
Change-type: patch
Signed-off-by: Rahul Thakoor <rahul@balena.io>

### CODE REVIEW

**Comment by rahul-thakoor** [comment]
@balena-ci retest

File: pages/learn/develop/integrations/aws.md [hunk 1]
```diff
@@ -34,10 +34,17 @@ Each `device` or `thing`, must have its own certificates that will be used to au
 
 ![generate certs](/img/integrations/aws/04_create_cert.png)
 
-In order to authenticate with the service, you will first need to download all four certificates from the dashboard to your computer (make sure you also download the `root CA` for AWS IoT).
+In order to authenticate with the service, you will first need to download the following files
+
+1. The thing's certificate
```
Comment (andrewnhem) [inline pages/learn/develop/integrations/aws.md:39]: I haven't tried building locally, but will this list populate with the correct numbers @rahul-thakoor ?

File: pages/learn/develop/integrations/aws.md [hunk 2]
```diff
@@ -34,10 +34,17 @@ Each `device` or `thing`, must have its own certificates that will be used to au
 
 ![generate certs](/img/integrations/aws/04_create_cert.png)
 
-In order to authenticate with the service, you will first need to download all four certificates from the dashboard to your computer (make sure you also download the `root CA` for AWS IoT).
+In order to authenticate with the service, you will first need to download the following files
```
Comment (andrewnhem) [inline pages/learn/develop/integrations/aws.md:37]: "files:" (add colon)

File: pages/learn/develop/integrations/aws.md [hunk 3]
```diff
@@ -46,16 +53,16 @@ Now it’s time to create some policies to allow our devices to communicate with
...
-| Effect       | Allow |
+| Action       | iot:\* |
+| ------------ | ------ |
+| Resource ARN | \*     |
+| Effect       | Allow  |
 
 ![create policy](/img/integrations/aws/07_add_policy.png)
 
-Go to **Secure -> Certificates*. Select the recently created certificate and attach both the thing and policy to it.
+Go to \*_Secure -> Certificates_. Select the recently created certificate and attach both the thing and policy to it.
```
Comment (garethtdavies) [inline pages/learn/develop/integrations/aws.md:63]: Not sure this change was intentional and results in a leading * - maybe some auto-formatting by the editor?

File: pages/learn/develop/integrations/aws.md (hunk 1 above)
Comment (rahul-thakoor) [reply pages/learn/develop/integrations/aws.md:39]: yes

File: pages/learn/develop/integrations/aws.md (hunk 3 above)
Comment (rahul-thakoor) [reply pages/learn/develop/integrations/aws.md:63]: yeah, I will fix. thanks @garethtdavies

File: pages/learn/develop/integrations/aws.md (hunk 2 above)
Comment (rahul-thakoor) [reply pages/learn/develop/integrations/aws.md:37]: got it. thanks @andrewnhem

**Comment by rahul-thakoor** [comment]
@balena-ci rebase

File: pages/learn/develop/integrations/aws.md [hunk 4]
```diff
@@ -67,42 +74,37 @@ At this point everything is ready on the AWS side, so let’s go ahead and confi
 
 ## Flashing the Raspberry Pi and deploying code
 
-### Step 1 - Set up the balenaCloud application
+### Using Python: MQTT Client example
 
-If you don’t have one already, [sign up for a balenaCloud account](https://dashboard.balena-cloud.com/signup). The first thing we need to do is to create a new application, for that click on **Create application**, give it a name and select a device type (on this example we will create a project called aws-iot to run on a Raspberry Pi 3).
+#### Set up the balenaCloud application
```
Comment (xginn8) [inline pages/learn/develop/integrations/aws.md:79]:
```suggestion
#### Set up the {{ $names.cloud.lower }} application
```

File: pages/learn/develop/integrations/aws.md [hunk 5]
```diff
@@ -67,42 +74,37 @@ At this point everything is ready on the AWS side, so let’s go ahead and confi
...
 
-With the application created, click on **Add device** and select the latest recommended balenaOS version, choose the network connection you desire, setup its credentials and download the balenaOS to your computer.
+If you don’t have one already, [sign up for a balenaCloud account](https://dashboard.balena-cloud.com/signup).
```
Comment (xginn8) [inline pages/learn/develop/integrations/aws.md:81]:
```suggestion
If you don’t have one already, [sign up for a {{ $names.cloud.lower }} account]({{ $links.dashboardUrl }}/signup).
```

File: pages/learn/develop/integrations/aws.md [hunk 6]
```diff
@@ -67,42 +74,37 @@ At this point everything is ready on the AWS side, so let’s go ahead and confi
...
 
-![add device](/img/integrations/aws/11_create_application.gif)
+You can deploy this project to a new balenaCloud application in one click using the button below:
```
Comment (xginn8) [inline pages/learn/develop/integrations/aws.md:83]:
```suggestion
You can deploy this project to a new {{ $names.cloud.lower }} application in one click using the button below:
```

File: pages/learn/develop/integrations/aws.md [hunk 7]
```diff
@@ -67,42 +74,37 @@ At this point everything is ready on the AWS side, so let’s go ahead and confi
...
-First of all, download the app from [the GitHub project repository]({{ $links.githubPlayground }}/balena-aws-iot-mqtt-example), and clone or download it to your computer.
+With the application created, click on **Add device** and select the latest recommended balenaOS version, choose the network connection you desire, setup its credentials and download the balenaOS to your computer.
 
-![github](/img/integrations/aws/14_github.png)
+![add device](/img/integrations/aws/11_create_application.gif)
 
+#### Flash your device
 
-Then, after [installing the balena CLI tools on your computer](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md), from the project directory, execute `balena push <appName>` where `<appName>` is the name of the application you created within the balenaCloud dashboard earlier. For this example project, we will use then `balena push aws-iot`.
+Use [balenaEtcher](https://www.balena.io/etcher/) to flash your Raspberry Pi with the downloaded OS image from the previous section. Insert the SD card into your computer, select the balenaOS image file, select the SD Card and click Flash!
```
Comment (xginn8) [inline pages/learn/develop/integrations/aws.md:99]:
```suggestion
Use [{{ $names.etcher.lower }}]({{ $links.etcherSiteUrl }}) to flash your Raspberry Pi with the downloaded OS image from the previous section. Insert the SD card into your computer, select the {{ $names.os.lower }} image file, select the SD Card and click Flash!
```

File: pages/learn/develop/integrations/aws.md [hunk 8]
```diff
@@ -67,42 +74,37 @@ At this point everything is ready on the AWS side, so let’s go ahead and confi
...
 
-If all went well you’ll see the balena unicorn mascot and the code you’ve just pushed will automatically be distributed to the devices in your application.
+![etcher](/img/integrations/aws/12_etcher.png)
 
-![balena push](/img/integrations/aws/15_balena_push.gif)
+After flashing is done, insert the SD card into your device and turn it on. After a few seconds, it should connect to the internet and show up on the balenaCloud dashboard.
```
Comment (xginn8) [inline pages/learn/develop/integrations/aws.md:103]:
```suggestion
After flashing is done, insert the SD card into your device and turn it on. After a few seconds, it should connect to the internet and show up on the {{ $names.cloud.lower }} dashboard.
```

File: pages/learn/develop/integrations/aws.md [hunk 9]
```diff
@@ -208,15 +210,15 @@ where you need to replace `KEYFILE` with the relevant filename (such as `xxxxxxx
...
+- [AWS SDK reference](https://aws.amazon.com/tools/#sdk), list of SDKs in all supported languages
+- [AWS IoT MQTT Client](https://console.aws.amazon.com/iot/home#/mqtt), useful for debugging AWS IoT communication
 
 ### Sample Apps
 
 A few sample apps to get started:
 
-* [balena-aws-lambda]({{ $links.githubPlayground }}/balena-aws-lambda) and [balena-aws-device]({{ $links.githubPlayground }}/balena-aws-device)
-* [balenaCloud AWS IoT MQTT Broker Example]({{ $links.githubPlayground }}/balena-aws-iot-mqtt-example) with full blog post for example:
+- [balena-aws-lambda]({{ $links.githubPlayground }}/balena-aws-lambda) and [balena-aws-device]({{ $links.githubPlayground }}/balena-aws-device)
```
Comment (xginn8) [inline pages/learn/develop/integrations/aws.md:223]: I believe these should be moved from `-playground` to `-examples` and updated accordingly.

Comment (rahul-thakoor) [reply pages/learn/develop/integrations/aws.md:223]: yes, they have been moved. GitHub does the redirect automatically. should we change it here too?

Comment (xginn8) [reply pages/learn/develop/integrations/aws.md:223]: I think that's best. I added the link stub here: https://github.com/balena-io/docs/pull/1445

**Comment by rahul-thakoor** [comment]
@balena-ci rebase

**Comment by rahul-thakoor** [comment]
hey @xginn8 I sent the PR to `master`. Should I change base to `staging` ?

**Comment by rahul-thakoor** [comment]
thanks @xginn8

**Pull request merged**